ironbound_chest.lua

```diff
@@ -78,38 +78,38 @@ minetest.register_node("castle_storage:ironbound_chest",{
 	allow_metadata_inventory_move = function(pos, from_list, from_index, to_list, to_index, count, player)
 		local meta = minetest.get_meta(pos)
 		if not has_ironbound_chest_privilege(meta, player) then
-			minetest.log("action", S("@1 tried to access a locked chest belonging to @2 at @3",
-				player:get_player_name(), meta:get_string("owner"), minetest.pos_to_string(pos)))
+			minetest.log("action", player:get_player_name() .. " tried to access a locked chest belonging to " ..
+				meta:get_string("owner") .. " at " ..  minetest.pos_to_string(pos))
 			return 0
 		end
 		return count
 	end,
 	allow_metadata_inventory_put = function(pos, listname, index, stack, player)
 		local meta = minetest.get_meta(pos)
 		if not has_ironbound_chest_privilege(meta, player) then
-			minetest.log("action", S("@1 tried to access a locked chest belonging to @2 at @3",
-					player:get_player_name(), meta:get_string("owner"), minetest.pos_to_string(pos)))
+			minetest.log("action", player:get_player_name() .. " tried to access a locked chest belonging to " ..
+				meta:get_string("owner") .. " at " .. minetest.pos_to_string(pos))
 			return 0
 		end
 		return stack:get_count()
 	end,
 	allow_metadata_inventory_take = function(pos, listname, index, stack, player)
 		local meta = minetest.get_meta(pos)
 		if not has_ironbound_chest_privilege(meta, player) then
-			minetest.log("action", S("@1 tried to access a locked chest belonging to @2 at @3",
-				player:get_player_name(), meta:get_string("owner"), minetest.pos_to_string(pos)))
+			minetest.log("action", player:get_player_name() .. " tried to access a locked chest belonging to " ..
+				meta:get_string("owner") .. " at " .. minetest.pos_to_string(pos))
 			return 0
 		end
 		return stack:get_count()
 	end,
 	on_metadata_inventory_move = function(pos, from_list, from_index, to_list, to_index, count, player)
-		minetest.log("action", S("@1 moves stuff in locked chest at @2", player:get_player_name(), minetest.pos_to_string(pos)))
+		minetest.log("action", player:get_player_name() " moves stuff in locked chest at " .. minetest.pos_to_string(pos))
 	end,
 	on_metadata_inventory_put = function(pos, listname, index, stack, player)
-		minetest.log("action", S("@1 moves stuff to locked chest at @2", player:get_player_name(), minetest.pos_to_string(pos)))
+		minetest.log("action", player:get_player_name() " moves stuff to locked chest at " .. minetest.pos_to_string(pos))
 	end,
 	on_metadata_inventory_take = function(pos, listname, index, stack, player)
-		minetest.log("action", S("@1 takes stuff from locked chest at @2", player:get_player_name(), minetest.pos_to_string(pos)))
+		minetest.log("action", player:get_player_name() " takes stuff from locked chest at " .. minetest.pos_to_string(pos))
```

Comment:
`castle_storage/ironbound_chest.lua:109: attempt to call a string value`

Why did luacheck not complain?